package.json

```diff
@@ -16,18 +16,21 @@
     "unicode"
   ],
   "main": "./lib",
+  "module": "./src/index.js",
```

Comment:
I think we'd probably want the module file to point to something that has passed through babel, to make sure that the only non-ES5 feature used is the import/export syntax.

Reply:
Ok, sure. I had got it into my head that Rollup took advantage of having access to the source for optimizations, but I guess I was mistaken.